ckanext/xloader/utils.py

```diff
@@ -243,6 +247,8 @@ def type_guess(rows, types=TYPES, strict=False):
         # element in case of a tie
         # See: http://stackoverflow.com/a/6783101/214950
         guesses_tuples = [(t, guess[t]) for t in types if t in guess]
+        if not guesses_tuples:
```

Comment:
How does this respond to files with no data?

Reply:
@ThrawnCA it won't get that far as Tabulator will fail because of empty data for the table. But as for empty cells, it works fine for that